### ISSUE TITLE
Insecure Loading of Google Fonts Bug

### ISSUE THREAD

**Issue by vestaxpdx** [reduxframework/redux-framework#3739]
Hi 

Since version 4.1.3 google fonts are not being rendered because they have been included via http not https

Edit* the only version that fixes this issue is all the way back to 3.6.18 so it seems since major version 4 there is an issue.

I hope this is an easy fix.

## Support Hash
Sorry link in comment does not work. Version: 4.1.3

## Steps to reproduce
Normal load of page.

## Expected Behavior

Fonts to load.

## Actual Behavior

Fonts don't load due to insecure content warning. They are being loaded via http not https

**Comment by dovy** [comment]
Wow, that is an easy fix. I'll get right on it.

**Comment by dovy** [comment]
Ok, super easy fix if you want to edit the code locally on your end. 
`~/redux-core/inc/fields/typography/class-redux-typography.php` line 840:
change:
```
return '//fonts.googleapis.com/css?family=' . $link;
```
to
```
return 'https://fonts.googleapis.com/css?family=' . $link;
```

I'll have a push out today that fixes this.  :)

**Comment by dovy** [comment]
Side note, this bug only manifests if your local site is not running HTTPS. Otherwise it will render properly.

**Comment by vestaxpdx** [comment]
Awesome thanks for the quick response. 

Hmm that is strange because the sites I noticed it broken on were all https as standard. I agree using // should be just fine and choose the right protocol. 

I guess hard coding https is no issue these days though if it fixes the issue.

**Comment by dovy** [comment]
So odd, I just saw the same thing on a secure site. So odd! The new fix being pushed now 4.1.4 solves it anyways. Thanks for the report.
